docs/development/_index.md

```diff
@@ -54,6 +54,109 @@ options can be appended to the main command as arguments.
 make image
 ```
 
+
+### Running the docker image
+
+With docker you can run the Headlamp image. Note, the mount arguments add folders that are referenced in the ~/.kube folders - you may need to add other folders if your config has more.
+
+```bash
+docker run --network="host" -p 127.0.0.1:4466:4466/tcp --mount type=bind,source="/home/rene/.minikube",target=$HOME/.minikube --mount type=bind,source="$HOME/.kube",target=/root/.kube quay.io/kinvolk/headlamp:latest /headlamp/server -html-static-dir /headlamp/frontend -plugins-dir=/headlamp/plugins
+```
+
+If you just made an new docker image you can run it like this:
+
+```bash
+$ DOCKER_IMAGE_VERSION=development make image
+...
+Successfully tagged kinvolk/headlamp:development
+
+$ docker run --network="host" -p 127.0.0.1:4466:4466/tcp --mount type=bind,source="/home/rene/.minikube",target=$HOME/.minikube --mount type=bind,source="$HOME/.kube",target=/root/.kube kinvolk/headlamp:development /headlamp/server -html-static-dir /headlamp/frontend -plugins-dir=/headlamp/plugins
+```
+
+Then go to https://localhost:4466
+
+
+### Minikube "incluster"
+
+This is if you want to use Headlamp incluster, and test it locally on minikube with a local docker image.
+
+#### docker image in the minikub docker environment
```

Comment:
We should try not to refer docker too much and rather use "Container image".

Reply:
I changed it to Container image in several places.